.changeset/funny-planes-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-satusehat': minor
```

Comment:
This should be a patch change because it doesn't add anything to the adaptor API. You can just rename `minor` to `patch` in the changeset here